certified-connectors/Morta/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13848 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Morta API Reference",
```

Comment:
There shouldn't be any reserved keywords, Please update title and description without keyword "API. 

Please refer this link https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters

Reply:
Fixed